contrib/storage-http/src/main/resources/bootstrap-storage-plugins.json

```diff
@@ -0,0 +1,10 @@
+{
+  "storage":{
+    "http" : {
+      "type":"http",
+      "connection": "https://api.sunrise-sunset.org/json",
```

Comment:
Do we want to put a dependency on an external system in this source code?

Reply:
Removed.